controller/scheduler/swarm.py

```diff
@@ -32,11 +43,15 @@ def create(self, name, image, command='', template=None, **kwargs):
             if mem[-2:-1].isalpha() and mem[-1].isalpha():
                 mem = mem[:-1]
         cpu = kwargs.get('cpu', {}).get(l['c_type'])
-        self.docker_cli.create_container(image=cimage, name=name,
-                                         command=command.encode('utf-8'), mem_limit=mem,
-                                         cpu_shares=cpu,
-                                         environment=[affinity],
-                                         host_config={'PublishAllPorts': True})
+        try:
+            self.docker_cli.create_container(image=cimage, name=name,
+                                             command=command.encode('utf-8'),
+                                             mem_limit=mem,
+                                             cpu_shares=cpu,
+                                             environment=[affinity],
+                                             host_config={'PublishAllPorts': True})
+        except Exception as err:
+            log_event(self.appid, str(err.message)+"  "+str(err.explanation), logging.ERROR)
```

Comment:
I think these errors come under errors.DockerException But just reading from exception object is a nicer way of logging the error